### ISSUE TITLE
Labelling data using pre-tranied model

### ISSUE THREAD

**Issue by ogulcankarayel5** [opencv/cvat#858]
I have 230k images. How can I label them using pre-trained model or any suggestion .

**Comment by benhoff** [comment]
There's integration with opencv's open model zoo and OpenVino: https://github.com/opencv/cvat/tree/develop/cvat/apps/auto_annotation

There's semantic segmentation via mask rcnn: https://github.com/opencv/cvat/tree/develop/components/auto_segmentation

There's semi automatic segmentation via Dextr: https://github.com/opencv/cvat/tree/develop/cvat/apps/dextr_segmentation 

OpenVino can convert almost any tensorflow and pytorch model.

Did that help? Is there any other questions I can answer?

**Comment by ogulcankarayel5** [comment]
I need tool for yolo format

**Comment by benhoff** [comment]
https://github.com/opencv/cvat/tree/develop/utils/open_model_zoo/yolov3

**Comment by nmanovic** [comment]
@ogulcankarayel5 ,

I hope @benhoff answered your questions. Please use our gitter channel to ask next questions. I will close the issue. 

Thanks @benhoff for your help!
